Configuration/Eras/python/Era_Run2_2016_cff.py

```diff
@@ -10,7 +10,8 @@
 from Configuration.Eras.Modifier_run2_L1prefiring_cff import run2_L1prefiring
 from Configuration.Eras.Modifier_pixel_2016_cff import pixel_2016
 from Configuration.Eras.Modifier_run2_jme_2016_cff import run2_jme_2016
+from Configuration.Eras.Modifier_strips_vfp30_2016_cff import strips_vfp30_2016
 
 Run2_2016 = cms.ModifierChain(run2_common, run2_25ns_specific,
- stage2L1Trigger, ctpps_2016, run2_HLTconditions_2016, run2_muon_2016, run2_egamma_2016, run2_L1prefiring, pixel_2016, run2_jme_2016)
+ stage2L1Trigger, ctpps_2016, run2_HLTconditions_2016, run2_muon_2016, run2_egamma_2016, run2_L1prefiring, pixel_2016, run2_jme_2016, strips_vfp30_2016)
```

Comment:
I'm confused why we still have modifications in this file.
Wasn't the strips_vfp30_2016 supposed to be removed completely?

Reply:
Hi @slava77 quoting my previous comment:
* the `strips_vfp30_2016` is kept only in order to persist exclusively in the Era_Run2_2016 the information if the APV simulation has been run or not (we can remove this modifier completely , depending on the outcome of the discussion at: #28355 (comment) )

Reply:
ah, thanks for clarifying.
But then it seems like this mod is not removed from Era_Run2_2017.
If the product is just a single float or int, it seems more straightforward to just always save it.

Reply:
thanks for catching. Addressed in https://github.com/cms-sw/cmssw/pull/28355/commits/b91964f7baa964fae3a1b96a6e541b50db5b3c28